initramfs/init_functions.sh

```diff
@@ -49,18 +49,34 @@ setup_usb_configfs() {
 	echo "rndis" > $CONFIGFS/g1/configs/c.1/strings/0x409/configuration \
 		|| echo "  Couldn't write configration name"
 
-	# Make sure the node for the eMMC exists
-	if [ -z "$(ls $EMMC)" ]; then
-		fatal_error "$EMMC could not be opened, possible eMMC defect"
+	if [[ $EMMC != "" ]]; then
```

Comment:
This is great but should probably be in it's own PR

Reply:
There is also functionality to get paella working at all. Maybe I can split it.